libs/react-helpers/src/lib/grid/size.tsx

```diff
@@ -26,11 +26,11 @@ export const Size = ({
   );
 };
 
-export interface ISizeCellProps extends ICellRendererParams {
+export interface AgSizeCellProps extends ICellRendererParams {
   value: number | string;
 }
 
-export const SizeCell = ({ value, data }: ISizeCellProps) => {
+export const AgSizeCell = ({ value, data }: AgSizeCellProps) => {
```

Comment:
chore (blocking): This doesn't appear to be used anywhere. Can we delete

Reply:
my plan was to use it in grids such as order list etc